src/AbstractArray.php

```diff
@@ -638,4 +638,23 @@ protected function sortingKeys(array &$elements, $order = SORT_ASC, $strategy =
                 ksort($elements, $strategy);
         }
     }
+
+    /**
+     * Returns the first value that matches $func condition
+     *
+     * @param Closure $func
+     *
+     * @return mixed The resulting value
+     */
+    public function find(Closure $func)
```

Comment:
Hm, it makes sense. It's like `filter()` but to get only a single value. I think it'll be nice.